charts/rancher-gke-operator/1.1.200/Chart.yaml

```diff
@@ -4,5 +4,5 @@ description: A Helm chart for provisioning GKE clusters
 home: https://github.com/rancher/gke-operator
 sources:
   - "https://github.com/rancher/gke-operator"
-version: 1.0.400
-appVersion: v1.1.1-rc10
+version: 1.1.200
```

Comment:
Needs to be updated here: https://github.com/rancher/rancher/issues/33528

Reply:
Updated version in linked issue.